Source/WebCore/page/FocusController.cpp

```diff
@@ -500,8 +497,9 @@ void FocusController::setFocused(bool focused)
 
 void FocusController::setFocusedInternal(bool focused)
 {
-    if (!isFocused())
-        focusedOrMainFrame().eventHandler().stopAutoscrollTimer();
+    RefPtr focusedOrMainFrame = this->focusedOrMainFrame();
```

Comment:
Can we put this inline the `if (!isFocused())` scope to avoid unnecessary ref counting churn when we're already focused?